x-pack/plugins/fleet/common/openapi/bundled.json

```
@@ -7671,8 +7671,8 @@
           },
           "namespace": {
             "type": "string",
-            "description": "namespace by default \"default\"",
-            "example": "default"
+            "description": "package policy namespace. It can be left blank to inherit agent policy's one",
```

```suggestion
            "description": "The package policy namespace. Leave blank to inherit the agent policy's namespace.",
```
I'd rephrase like this (and of course the same for the other descriptions).

---

Fixed!